src/popup/components/TransactionLabel.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <div class="transaction-label">
+    <PendingIcon
+      v-if="isPending"
+      class="icon"
+    />
+    <div
+      v-else-if="isErrorTransaction"
+      class="error"
+    >
+      <WarningIcon
+        v-if="tx.returnType === 'abort'"
+        class="icon"
+      />
+      <RevertedIcon
+        v-else
+        class="icon"
+      />
+    </div>
+
+    <div class="label">
+      <span v-if="label.customPending && isErrorTransaction">
+        {{ label.customPending }}
+      </span>
+      <span
+        v-else-if="!isPending"
+        class="type"
+        :class="{ secondary: transactionOwner }"
+      >
+        {{ label.text }}
+      </span>
+      <span
+        v-if="isErrorTransaction"
+        class="error-type"
+      >
+        {{ $t('transaction.returnType')[tx.returnType] }}
+      </span>
+      <span
+        v-if="isPending"
+        :class="{ secondary: !label.customPending || transactionOwner }"
+      >
+        {{ label.customPending || $t('transaction.type.pending') }}...
+      </span>
+      <span
+        v-else-if="!transactionOwner"
+        class="secondary"
+      >{{ transactionDate }}</span>
+
+      <span v-if="transactionOwner">
+        <span class="secondary">{{ $t('by') }}</span>
```

Comment:
In translation files there is no root `by` item. Please add one. And also please put it under the `common` key so it would be `common.by`.